aries-core/src/js/components/core/Selector/Selector.js

```diff
@@ -50,7 +50,7 @@ const Selector = ({
         pad={pad}
         round={round}
         selected={selected}
-        gap="xsmall"
+        gap="3xsmall"
```

Comment:
this looks good I see above though 
https://github.com/grommet/hpe-design-system/pull/5281/files#diff-238cb28c0edb3bec4253bb8f2b8665c547c22fcdaa7ab7de9df8e24fdd1a4732R31

it did not catch the `pad`

Reply:
this is fixed